static/js/containers/admin/EditChannelModeratorsPage_test.js

```diff
@@ -113,155 +176,190 @@ describe("EditChannelModeratorsPage", () => {
     })
 
     it("renders the form", async () => {
-      const wrapper = await renderPage()
-      const props = wrapper.find("EditChannelMembersForm").props()
+      const { inner } = await render()
+      const props = inner.find("EditChannelMembersForm").props()
       assert.deepEqual(props.memberTypeDescription, "moderator")
       assert.deepEqual(props.form, newMemberForm())
     })
 
-    it("tries to add a new moderator but fails validation", async () => {
-      const wrapper = await renderPage()
-      await listenForActions([actions.forms.FORM_VALIDATE], () => {
-        wrapper
-          .find("form")
-          .props()
-          .onSubmit({ preventDefault: helper.sandbox.stub() })
-      })
+    it("updates the email", async () => {
+      const { inner, store } = await render()
+      const props = inner.find("EditChannelMembersForm").props()
+      const email = "new@email.com"
+      props.onUpdate({ target: { name: "email", value: email } })
 
-      assert.equal(
-        helper.store.getState().forms["channel:edit:moderators"].errors.email,
-        "Email must not be blank"
-      )
+      const actions = store.getActions()
+      assert.deepEqual(actions[actions.length - 1], {
+        type:    FORM_UPDATE,
+        payload: {
+          formKey: MODERATORS_KEY,
+          value:   { email }
+        }
+      })
     })
 
     it("tries to add a new moderator but the API request fails", async () => {
-      const wrapper = await renderPage()
-      const email = "new@email.com"
-
-      helper.addChannelModeratorStub.returns(Promise.reject())
+      helper.addChannelModeratorStub.returns(Promise.reject("an error"))
 
-      wrapper
-        .find("input[name='email']")
-        .props()
-        .onChange({
-          target: { name: "email", value: email }
-        })
-      await listenForActions(
-        [
-          actions.channelModerators.post.requestType,
-          actions.channelModerators.post.failureType,
-          actions.forms.FORM_VALIDATE,
-          actions.forms.FORM_VALIDATE
-        ],
-        () => {
-          wrapper
-            .find("form")
-            .props()
-            .onSubmit({ preventDefault: helper.sandbox.stub() })
+      const email = "new@email.com"
+      const { inner, store } = await render({
+        forms: {
+          [MODERATORS_KEY]: {
+            value: {
+              email
+            },
+            errors: {}
+          }
         }
-      )
+      })
 
+      const props = inner.find("EditChannelMembersForm").props()
+      props.onSubmit({ preventDefault: helper.sandbox.stub() })
+
+      // let promise resolve
+      await wait(0)
```

Comment:
can you just `await props.onSubmit(...)` here?

Reply:
Sure, just made the change